public/app/features/explore/RichHistory/RichHistoryQueriesTab.tsx

```diff
@@ -211,6 +211,7 @@ export function RichHistoryQueriesTab(props: Props) {
           )}
           <div className={styles.filterInput}>
             <FilterInput
+              escapeRegex={false}
```

Comment:
The same component is used in `RichHistoryStarredTab` so probably also needs to be updated.

It's interesting that this prop is set to `true` by default. @grafana/user-essentials is this component meant to be used in a specific context or mainly when regex is used for search? Should we use different components for filtering? (cc: @grafana/design-system). It's quite easy to forget to set that prop and there might be similar bugs in other places.

I was thinking we could use `<Input prefix={<Icon name="search" />} />` but it doesn't have that nice `X Clear` button :(

Just for context if someone's interested. This component was added to fix https://github.com/grafana/grafana/issues/12972 in https://github.com/grafana/grafana/pull/15382 and the escaping logic was always applied. It was changed to be optional to fix https://github.com/grafana/grafana/issues/55312 in #55314

Reply:
Thanks @ifrost. I fixed it in [0648b54](https://github.com/grafana/grafana/pull/64879/commits/0648b5462b090dea1f0086747878199528b373a0).

Btw, do you consider this a bug that should be backported?

Reply:
Thanks! It is a bug but I think it should not be backported.